cl/search/models.py

```diff
@@ -2907,6 +2907,28 @@ class Meta:
         unique_together = ("citing_opinion", "cited_opinion")
 
 
+class OpinionsCitedByRECAPDocument(models.Model):
+    citing_document = models.ForeignKey(
+        RECAPDocument, related_name="cited_opinions", on_delete=models.CASCADE
+    )
+    cited_opinion = models.ForeignKey(
+        Opinion, related_name="citing_documents", on_delete=models.CASCADE
+    )
+    depth = models.IntegerField(
+        help_text="The number of times the cited opinion was cited "
+        "in the citing document",
+        default=1,
+        db_index=True,
```

Comment:
Sorry, I just caught one more thing. This [should be done in the `indexes` meta attribute](https://github.com/django/django/pull/12716). It's easier to adjust and manage when it's there. Probably I should teach semgrep to throw an alert when db_index is used...someday. 

Anyhow, mind making that last tweak, and then we'll be good to go?

Reply:
And in case it's not obvious, after you do this, the SQL migration and the migration file both probably need regeneration.